src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
+BuildRequires:  R-devel >= 3.1.0
```

Comment:
Suppose uses have R = 3.x (which is higher than 3.1.0) installed with yum. Since we don't try it out in hawq, what if below two potential issue:
1) plr can not be built and installed due to API change in R
2) plr can be build and installed successfully, but have runtime issue